lib/sandbox.js

```diff
@@ -1290,12 +1290,13 @@ function sandBox(script, name, verbose, debug, context) {
                 return;
             }
 
-            sandbox.__engine.__schedules += 1;
-
             if ((typeof pattern === 'string' && pattern[0] === '{') || (typeof pattern === 'object' && pattern.period)) {
                 sandbox.verbose && sandbox.log(`schedule(wizard=${typeof pattern === 'object' ? JSON.stringify(pattern) : pattern})`, 'info');
                 const schedule = context.scheduler.add(pattern, sandbox.name, callback);
-                schedule && script.wizards.push(schedule);
+                if (schedule) {
+                    script.wizards.push(schedule);
+                    sandbox.__engine.__schedules += 1;
```

Comment:
I think this is needed also for other places in the function, but you are riught there are returns without a new schedule added that do not decrease it

Reply:
i checked tis, `sandbox.__engine.__schedules` is only decreased at other places and only if > 0, so this should be fine

Reply:
In fact this is an counter to show the userhow many schedules were set by a script on start ... so we should have it correct as number also for ther cases. But I can adjust that after merge too